lib/mucore.mli

```diff
@@ -278,6 +278,7 @@ type parse_ast_label_spec =
   { label_spec : (Sym.t, Cerb_frontend.Ctype.ctype) Cerb_frontend.Cn.cn_condition list }
 
 type 'TY label_def =
+  | Other of Locations.t * Sym.t * Cerb_frontend.Annot.label_annot * unit arguments
```

Comment:
Could you add a comment here and in the `.ml` file explaining that this is needed for skipping label inlining for testing?